packages/blocks/callout/src/EmojiIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {
+  CSSProperties,
+  MouseEventHandler,
+  VoidFunctionComponent,
+} from "react";
+
+export interface EmojiIconProps {
+  value: string | undefined;
+  onChange: (value: string | undefined) => void;
+}
+
+const variants = ["📢", "💡", "🤔", "👉", "📣", "📌", "🚧"];
+
+const wrapperStyle: CSSProperties = {
+  display: "inline-block",
+  width: "1.5em",
+  left: "0.25em",
+  height: "1.5em",
+  cursor: "pointer",
+  textAlign: "center",
+  padding: 0,
+  position: "absolute",
+  userSelect: "none",
+  border: "none",
+  background: "none",
+  overflow: "hidden",
+};
+
+export const EmojiIcon: VoidFunctionComponent<EmojiIconProps> = ({
+  value,
+  onChange,
+}) => {
+  const handleDivClick: MouseEventHandler = (event) => {
+    event.stopPropagation();
+    onChange(variants[(variants.indexOf(value ?? "") + 1) % variants.length]);
+  };
```

Comment:
I'm getting a lot of console errors regarding the callout button here
```
A component is `contentEditable` and contains `children` managed by React. It is now your responsibility to guarantee that none of those nodes are unexpectedly modified or duplicated. This is probably not intentional.
button
```
I'm not sure if these can be ignored or not

Reply:
Ticket for this is here https://app.asana.com/0/1201707629991380/1201837405043820/f (internal)